setup.py

```diff
@@ -84,9 +84,9 @@ def get_extensions():
     return extensions
 
 
-install_requires = []
+install_requires = ['torch']
```

Comment:
I still have my problems with this, but mostly only due to bad experience in the past. For example, it is impossible for me to install the [`sentence-transformers`](https://github.com/UKPLab/sentence-transformers/blob/master/setup.py) library since due to whatever reasons PyTorch will always get re-installed.

It also looks like `pip install torch-scatter` will still fail when no PyTorch version is installed, right?

Reply:
I will see if I can replicate this problem using some clean virtual environments